py3status/modules/xsel.py

```diff
@@ -1,14 +1,20 @@
 # -*- coding: utf-8 -*-
+import re
+import time
 """
 Display X selection.
 
 Configuration parameters:
     cache_timeout: refresh interval for this module (default 0.5)
-    command: the clipboard command to run (default 'xsel -o')
+    command: the clipboard command to run (default 'xsel -output')
+    click_command: the clipboard command to run on click
+        (default 'xsel --clear --clipboard')
     format: display format for this module (default '{selection}')
     max_size: strip the selection to this value (default 15)
     symmetric: show beginning and end of the selection string
         with respect to configured max_size. (default True)
+    log_file: where to log the clipboard, default doesn't log at all
```

Comment:
(Opinion) Avoid `default doesn't log at all`. Use `specify the clipboard log to use (default None)`